packages/cardano-services/src/Utxo/DbSyncUtxoProvider/util.ts

```diff
@@ -0,0 +1,5 @@
+import { Cardano } from '@cardano-sdk/core';
+
+export const generateAssetId = (policy: string, name: string) => Cardano.AssetId(policy + name);
```

Comment:
Anything generic like this should be defined in `core/src/Cardano/util`, but in this case as we already have: https://github.com/input-output-hk/cardano-js-sdk/blob/dc33f153288255b256453209433f08dae1a22291/packages/core/src/Asset/util/assetId.ts#L10

... I suggest this be removed, in favour of:

Reply:
If I use this function I have to cast both name and policy to emurgo types which doesnt look good. I suggest to create a new function with the current `generateAssetId` implementation in `core` package named `createAssetIdFromCore`

Reply:
Right, I misinterpreted the param types. We'd be better renaming the existing function to `createAssetIdFromCSL`, so that the current util implies it uses core types, but I'd rather you not get bogged down. Let's leave it here for the time being